src/Microsoft.Health.Fhir.Shared.Api/Controllers/ValidateController.cs

```diff
@@ -14,48 +15,88 @@
 using Microsoft.Health.Fhir.Api.Features.Filters;
 using Microsoft.Health.Fhir.Api.Features.Routing;
 using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
 using Microsoft.Health.Fhir.Core.Messages.Operation;
+using Microsoft.Health.Fhir.Core.Models;
 using Microsoft.Health.Fhir.ValueSets;
 
 namespace Microsoft.Health.Fhir.Api.Controllers
 {
     [ServiceFilter(typeof(AuditLoggingFilterAttribute))]
     [ServiceFilter(typeof(OperationOutcomeExceptionFilterAttribute))]
     [ServiceFilter(typeof(ValidateContentTypeFilterAttribute))]
-    [ServiceFilter(typeof(ValidationQueryFilterAndParameterParserAttribute))]
```

Comment:
Just for my understanding - This filter was doing some validation on the mode. Since this has been removed, is there a reason we no longer need to validate the mode information.

Reply:
Pretty much.

---
In reply to: [581633730](https://github.com/microsoft/fhir-server/pull/1654#discussion_r581633730) [](ancestors = 581633730)